/// <reference types="@sveltejs/kit" /> isn't work as intended.

### Describe the bug

In the latest version of sveltekit reference type declarations in the global.d.ts file isn't work as intended.

When I run ``tsc --noEmit`` it will get back with the following errors:
```
src/lib/env.ts:1:37 - error TS2339: Property 'env' does not exist on type 'ImportMeta'.

1 export const BASE_URL = import.meta.env.VITE_BASE_URL;
                                      ~~~
src/lib/utils/api.ts:3:22 - error TS2307: Cannot find module '$app/navigation' or its corresponding type declarations.

3 import { goto } from '$app/navigation';
```

Extending types with these types will solve the problem.
I think would be better using the references rather than this.
```
// tsconfig.json
{
  "compilerOptions": {
    "types": ["vite/client", "./node_modules/@sveltejs/kit/types/ambient-modules"]
  }
 }
```

If I remove these options and /// <reference types="@sveltejs/kit" />  from the global.d.ts, the result will be the same.
AFAIK all of the added compilerOpt types is imported & exported in @sveltejs/kit, but isn't works as it has to.

### Reproduction

Install a fresh svelte kit template and turn into typescript.
Update packages.
Try using env declarations.
run ``tsc --noEmit``

### Logs

```shell
src/lib/env.ts:1:37 - error TS2339: Property 'env' does not exist on type 'ImportMeta'.

1 export const BASE_URL = import.meta.env.VITE_BASE_URL;
                                      ~~~

src/lib/utils/api.ts:3:22 - error TS2307: Cannot find module '$app/navigation' or its corresponding type declarations.

3 import { goto } from '$app/navigation';
```
```


### System Info

```shell
System:
    OS: Linux 5.11 Ubuntu 20.04.3 LTS (Focal Fossa)
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 1.66 GB / 15.48 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 16.13.0 - /usr/bin/node
    npm: 8.1.0 - /usr/bin/npm
  Browsers:
    Chrome: 96.0.4664.45
    Firefox: 94.0
  npmPackages:
    @sveltejs/adapter-static: ^1.0.0-next.21 => 1.0.0-next.21 
    @sveltejs/kit: ^1.0.0-next.196 => 1.0.0-next.196 
    svelte: ^3.44.2 => 3.44.2
```


### Severity

serious, but I can work around it

### Additional Information

Extending types with these types will solve the problem.
I think would be better using the references rather than this.
```
{
  "compilerOptions": {
    "types": ["vite/client", "./node_modules/@sveltejs/kit/types/ambient-modules"]
  }
 }
```

--- a/.changeset/beige-eyes-poke.md
+++ b/.changeset/beige-eyes-poke.md
@@ -1,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+Fix types reference in exports in package.json

--- a/packages/kit/package.json
+++ b/packages/kit/package.json
@@ -68,6 +68,9 @@
 	},
 	"exports": {
 		"./package.json": "./package.json",
+		".": {
+			"types": "./types/index.d.ts"
+		},
 		"./ssr": {
 			"import": "./dist/ssr.js"
 		},
@@ -79,8 +82,7 @@
 		},
 		"./install-fetch": {
 			"import": "./dist/install-fetch.js"
-		},
-		"./types": "./types/index.d.ts"
+		}
 	},
 	"types": "types/index.d.ts",
 	"engines": {

